glide.yaml

```diff
@@ -69,3 +69,13 @@ import:
   version: v0.8.0
   subpackages:
   - prometheus/promhttp
+- package: gopkg.in/yaml.v2
```

Comment:
This dependency is defined in glide.yaml in coreos/etcd with version cd8b52f8269e0feb286dfeef29f8fe4d5b397e0b (the same I got when I tried) and we definitely should not change the version here. Glide flattens this package to the same version every time so I think this does not need to be here. 

Others seem ok to me

Reply:
Ok, removed it